### ISSUE TITLE
How to use configuration from the parent folder?

### ISSUE THREAD

**Issue by denysvarchenko** [realm/SwiftLint#2887]
Hello,

I have read all the documentation but haven't found the mention about it.
I have a project that is combined with .podspec from several projects. So the structure is:
- Project folder
      - CoreProject
      - SubProject1(actually one of the submodules of the app)
      - SubProject1
      ...
      - MainProject: It is been compiled at the end and distributed to the end-users

For each project, I have created a script that runs swift lint, and swift lint shows style errors properly. But I needed to add swiftlint.yml configuration to each folder. I would like to use one configuration file for all projects. Is it actually possible to set the folder where the script should search for the configuration?
      
Thank you

**Comment by vani2** [comment]
I do that way. I have several targets, each of them has a folder with different `.swiftlint.yml` files. So in that target build phases I call: 
`$SRCROOT/Pods/SwiftLint/swiftlint --path $SRCROOT/Modules/FolderOne`,
`$SRCROOT/Pods/SwiftLint/swiftlint --path $SRCROOT/Modules/FolderTwo` and so on.

**Comment by denysvarchenko** [comment]
> I do that way. I have several targets, each of them has a folder with different `.swiftlint.yml` files. So in that target build phases I call:
> `$SRCROOT/Pods/SwiftLint/swiftlint --path $SRCROOT/Modules/FolderOne`,
> `$SRCROOT/Pods/SwiftLint/swiftlint --path $SRCROOT/Modules/FolderTwo` and so on.

Thank you. This is a pretty good approach. I would like to have only on .swiftlint.yml, but I think I can handle my problem with this

**Comment by denysvarchenko** [comment]
I found the solution that also fixes my problem.
I have created the  `.swiftlint.yml` in the root. Included everything in the folders on the same level and excluded for each folder specific Pod libraries
`included:
  - ../
excluded: # paths to ignore during linting. Takes precedence over `included`.
  - ../*/Pods/SQLClipher
  - ../*/Pods/PromiseKit
  - ../*/Pods/UICircularProgressRing`

For each subproject folder, I created a symbolic link, but it is important that path in this link is `../.swiftlint.yml`, not a full path, because otherwise other developers will not be able to use it.

Finally, I have enabled symlinks for my git repo(otherwise they will be broken after checkout on a new computer
